src/promptflow/tests/executor/unittests/_core/test_tools_manager.py

```diff
@@ -159,6 +159,14 @@ def test_collect_package_tools_if_node_source_tool_is_legacy(self):
         package_tools = collect_package_tools(legacy_node_source_tools)
         assert "promptflow.tools.azure_content_safety.analyze_text" in package_tools.keys()
 
+    def test_collect_package_tools_set_defaut_input_index(self, install_promptflow_tools_pkg):
```

Comment:
As a test in executor. It would be better to use a mocked tool instead of the tools in promptflow-tools.

Reply:
Updated test, thanks.